lib/test_data/account_creator.rb

```diff
@@ -0,0 +1,31 @@
+class TestData::AccountCreator
+  DATA_FILE = 'tmp/test_data_account_ids.txt'.freeze
+
+  def self.create!(id)
+    company_name = generate_company_name
+    domain       = generate_domain(company_name)
+    account      = Account.create!(
+      id: id,
+      name: company_name,
+      domain: domain,
+      created_at: Faker::Time.between(from: 2.years.ago, to: 6.months.ago)
+    )
+    persist_account_id(account.id)
```

Comment:
You can use the `internal_attributes` field on the account model also instead of a text file @pranavrajs @sojan-official wdyt?

Reply:
The current way is fine too, just a nitpick, feel free to ignore